tests/10-EdvClient.spec.js

```diff
@@ -64,6 +64,67 @@ describe('EdvClient', () => {
     docs.length.should.equal(1);
   });
 
+  it('find a document using a multi property query', async () => {
+    const client = await mock.createEdv();
+    client.ensureIndex({attribute: 'content.indexedKey'});
+    client.ensureIndex({attribute: 'content.someKey'});
+
+    const testId = await EdvClient.generateId();
+    const doc = {
+      id: testId,
+      content: {
+        someKey: {
+          b: 5,
+          a: 4
+        },
+        indexedKey: 'value1'
+      }
+    };
+
+    await client.insert({
+      doc,
+      invocationSigner,
+      keyResolver
+    });
+
+    // it should find the document when property keys are in same order.
+    const docs = await client.find({
+      equals: {'content.someKey': {
+        b: 5,
+        a: 4
+      }},
```

Comment:
This looks like our linter may not be catching an error here. @mattcollier, thoughts?